packages/vite/package.json

```diff
@@ -79,7 +79,6 @@
     "@vue/compiler-dom": "^3.1.5",
     "acorn": "^8.4.1",
     "acorn-class-fields": "^1.0.0",
-    "acorn-numeric-separator": "^0.3.6",
```

Comment:
**question:** Why is this not required anymore? What does it has to do with this PR?
Or is it really just used at these lines and can therefore be removed?

Reply:
see commit message:

> fix: remove dependency acorn-numeric-separator (#4782)
>
> deprecated acorn-numeric-separator@0.3.6: acorn>=7.4 supports numeric separators